landlab/components/flow_accum/flow_accumulator.py

```diff
@@ -732,6 +753,70 @@ def node_order_upstream(self):
         """Return the upstream node order (drainage stack)."""
         return self._grid["node"]["flow__upstream_node_order"]
 
+    @property
+    def link_order_upstream(self):
```

Comment:
Since this isn't something being saved in the grid, I wonder if it should be a function rather than a property.

Reply:
This makes sense. Will do.

Reply:
I also determined that a few other new weren't really properties (so I fixed them too).